test/StoragelessTest/Session/ArrayAccessorTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types = 1);
+
+namespace PSR7SessionsTest\Storageless\Session;
+
+use PHPUnit_Framework_TestCase;
+use PSR7Sessions\Storageless\Session\LazySession;
+use PSR7Sessions\Storageless\Session\ArrayAccessor;
+use PSR7Sessions\Storageless\Session\SessionInterface;
+
+/**
+ * @covers \PSR7Sessions\Storageless\Session\ArrayAccessor
+ */
+final class SessionArrayAccessorTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var SessionInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $wrappedSession;
+
+    /**
+     * @var callable|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $sessionLoader;
+
+    /**
+     * @var ArrayAccessor
+     */
+    private $arrayAccess;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function setUp()
+    {
+        $this->wrappedSession = $this->createMock(SessionInterface::class);
+        $this->sessionLoader = $this->getMockBuilder(\stdClass::class)->setMethods(['__invoke'])->getMock();
+        $session = LazySession::fromContainerBuildingCallback($this->sessionLoader);
+
+        $this->arrayAccess = new ArrayAccessor($session);
+    }
+
+    public function testIsAnSessionArrayAccessor()
+    {
+        self::assertInstanceOf(ArrayAccessor::class, $this->arrayAccess);
```

Comment:
@SvenRtbg just a quick copy of the other test.  (so it is in there as well). I'll take it out.